crates/iota-node/Cargo.toml

```diff
@@ -23,6 +23,7 @@ fastcrypto.workspace = true
 fastcrypto-zkp.workspace = true
 futures.workspace = true
 humantime.workspace = true
+iota-sdk-types = { workspace = true, features = ["serde"] }
```

Comment:
Do we really need the `serde` feature here? Same for other toml which we enable the `serde`.